ptx/sec_deriv_inverse_function.ptx

```diff
@@ -378,7 +399,12 @@
   <sidebyside widths="47% 47%" margins="0%">
       <!-- START figures/fig_inverse4.tex -->
       <image xml:id="img_inverse4">
-        <description></description>
+      <shortdescription>graph of sin x and sin^{-1} x with corresponding tangent lines</shortdescription>
```

Comment:
I think it would be better to avoid sin^{-1} in the short description. This will be read by the screen reader as something like: "sin carat left brace hyphen 1 right brace".
(And 'sin' will probably rhyme with 'tin'.)

You could try, "Graphs of the sine function and its inverse, with corresponding tangent lines."